packages/logging/src/services/logger/logger.service.ts

```
@@ -96,76 +112,44 @@ export class LoggerService implements Logger {
     return this;
   }
 
-  log(message: any): void {
-    return this.pino.info(message);
+  log(message: LogMessage): void {
+    return this.info(message);
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)